recipes/ipyrad/meta.yaml

```diff
@@ -33,11 +33,11 @@ requirements:
     - requests
     - cutadapt
     - pysam >=0.15
-    - mpi4py >=3.0
```

Comment:
You could also use a selector here so the dependency is automatically installed for python 3.7: `[ py == 37 ]` or `[ py >= 37 ]` depending on what you need.
Example:
```
    - cutadapt
    - pysam >=0.15
    - mpi4py >=3.0  # [ py >= 37 ]
```